src/MensaWidget/MWApiWrapperTest.class.st

```diff
@@ -0,0 +1,62 @@
+Class {
+	#name : #MWApiWrapperTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'apiWrapper'
+	],
+	#classInstVars : [
+		'apiWrapper'
+	],
+	#category : #'MensaWidget-Tests'
+}
+
+{
+	#category : #environment,
+	#'squeak_changestamp' : 'Oscar 5/22/2024 22:54'
+}
+MWApiWrapperTest >> setUp [
+	apiWrapper := MWApiWrapper new 
+]
+
+{
+	#category : #environment,
+	#'squeak_changestamp' : 'Oscar 5/26/2024 09:26'
+}
+MWApiWrapperTest >> tearDown [
+	apiWrapper  := nil. 
+]
+
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'Oscar 5/26/2024 16:17'
+}
+MWApiWrapperTest >> testApiCallWithInvalidUrl [
+    | responseWrapper errorOccurred |
+    errorOccurred := false.
+    [
+        responseWrapper := apiWrapper getMealsFrom: '404' on: '2002-05-06'.
+    ] on: NetworkError do: [ :ex |
+        errorOccurred := true.
+    ].
```

Comment:
im not sure how it is supposed to like on this one but usually the brakets start at the first line and end with the last line
@Kangaroo6942

Reply:
yeah, it would be ...true].
Also, no space is needed after "[", so it would be [:ex ...
And the last statement (which at the same time is the only one) does not need a dot at the end

Reply:
we changed this we now need spaces before and after a square bracket